### DISCUSSION TITLE
Bookworm Translations

### DISCUSSION THREAD

**Discussion by mush42** [mush42/bookworm#61] [General]
Hello Bookworm Translators:

The first beta of Bookworm v0.4 has just been released.

https://getbookworm.com/blog/post/v0.4b1/

During this phase we want to hunt down any remaining bugs, and complete the translation work before the final version which is scheduled for the 30th of June.

If you're an existing or new translator, please grab the .pot translation template to update your translation catalog. Get the .pot file from the release assets here:

https://github.com/blindpandas/bookworm/releases/tag/untagged-6df07aa781ee93de60e3

After completing your translation work, please upload the final catalog with a comment in this discussion.

Best Regards,

**Comment by cary-rowen** [comment]
Hello! Selecting the Simplified Chinese interface language does not seem to change.

**Comment by k-kolev1985** [comment]
Hello,

Here is the updated bulgarian translation for Bookworm 0.4.

You might consider fixing the following english string:
> Could not open file\n
> .Either the file  has been damaged during download, or it has been corrupted in some other way.

The period symbol should be on the end of the first line and not at the beginning of the second one.

BTW: From where can we get the updated english user guide, so we can update our localized user guides?

Thanks in advance!

[bg_bookworm.zip](https://github.com/blindpandas/bookworm/files/6589776/bg_bookworm.zip)
